src/js/setup-text-tracks.js

```diff
@@ -0,0 +1,136 @@
+import dashjs from 'dashjs';
+
+/*
+ * Attach text tracks from dash.js to videojs
+ *
+ * @param {videojs} player the videojs player instance
+ * @param {array} tracks the tracks loaded by dash.js to attach to videojs
+ *
+ * @private
+ */
+function attachDashTextTracksToVideojs(player, tracks) {
+  const mediaPlayer = player.dash.mediaPlayer;
+
+  const trackDictionaryKeyedByTextTrack = new WeakMap();
```

Comment:
We might not want to either Map either depending on which browsers we're targeting for this project. We wouldn't want to require es6-shim for `Map()` support if we expect to be able to run this package in a browser that supports MSE+EME but not `Map()`.

@forbesjo do need to worry about this?